index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css\style.css"/>
+<body>
+ <div id="header"><h1 id="title">Microverse Survey Form</h1>
+  <p id="description">It's all because of you that we got this far.Thank you!</p>
+ </div>
+  <form id="survey-form">
+   <label id="name-label">Name</label>
+   <br>
+   <input id="name" type="text" placeholder="Enter your name" required>
+   <br>
+   <label id="email-label" >Email</label>
+   <br>
+   <input id="email" type="email" placeholder="Enter your email" required>
+   <br>
+   <label id="number-label">Age</label>
+   <br>
+     <input id="number" type="number" placeholder="Enter your age" min=15 max=125 >
+     <br>
+     <label>What's your occupation</label>
+     <br>
+     <select id="dropdown">
+       <option> Choose from below</option> 
+       <option>Software Engineer</option>
+       <option>Teacher</option>
+       <option>Banker</option>
+       <option>Student</option>
+     </select>
+     <br>
+     <label>Are you willing to work with Microverse</label>
+     <br>
+     <input value="Absolutely" name="decision" type="radio"> 
+     <label>Absolutele</label>
```

Comment:
I kindly suggest that you check the spelling of all tags that will be visible on your site. Line 37, "Absolutely" is spelled wrong Mostly when readers notice lots of grammatical errors on a website, they don't take whatever is on the site seriously.